config/cors.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * The `corsOptions` object is a configuration object that tells the server which origins are
+allowed to make requests to this server. 
+ * 
+ * The `origin` property is an array of allowed origins. 
+ * 
+ * The `optionsSuccessStatus` property is a number that tells the browser how to respond to a CORS
+OPTIONS request. 
+ * 
+ * The `credentials` property is a boolean that tells the browser whether to send credentials
+(cookies, for example) with the request. 
+ * 
+ * The `exposedHeaders` property is an array of headers
+ */
+export const corsOptions = {
+  origin: ["http://localhost:3000", "https://socialhq-api.up.railway.app/"],
```

Comment:
Just make a note here to update this with vercel's URL
you may forget it in future

Reply:
> Just make a note here to update this with vercel's URL you may forget it in future

The local host part, right ?

Reply:
> > Just make a note here to update this with vercel's URL you may forget it in future
> 
> The local host part, right ?

its an array of possible hosts that can send requests to your server

so you can just add it like this 
```
['https://myapp.vercel.app', 'http://localhost:3000', 'some other domain .com etc']
```